src/lib/styles/variables.less

```diff
@@ -0,0 +1,21 @@
+@black: #000;
+@charcoal: #252525;
+@grey-dark: #343434;
+@grey-dove: #676767;
+@grey: #818181;
+@grey-silver: #a3a3a3;
+@grey-light: #ddd;
+@white: #fff;
+@white-smoke: #f0f0f0;
+@white-fog: #e6e6e6;
+@blue: #368aa2;
+@blue-emphasis: #2b7187;
+@blue-muted: #708e97;
+@green: #61A85D;
+@orange: #f59f51;
+@red: #ee4d3e;
+@red-emphasis: #ff0000;
+
+@padding-base: 10px;
```

Comment:
there will be more work done as we move along to incorporate variables into existing styles, still coming together a little bit